apps/sv/frontend/src/components/governance/ConfigValuesChanges.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Box, Typography } from '@mui/material';
+import { ConfigChange } from '../../utils/types';
+import { PartyId } from '@lfdecentralizedtrust/splice-common-frontend';
+
+interface ConfigValuesChangesProps {
+  changes: ConfigChange[];
+}
+
+export const ConfigValuesChanges: React.FC<ConfigValuesChangesProps> = props => {
```

Comment:
built this a while back and used it in the proposal details page. Just extracting it to its own file here for better visibility